pkg/services/dashboards/database/database.go

```diff
@@ -23,8 +25,11 @@ type DashboardStore struct {
 	log      log.Logger
 }
 
-// DashboardStore implements the Store interface
-var _ dashboards.Store = (*DashboardStore)(nil)
+var (
+	// DashboardStore implements the Store interface
+	_       dashboards.Store = (*DashboardStore)(nil)
+	dialect migrator.Dialect
```

Comment:
You can use it for now, later on we need to pass DB.db to initialize the stores to avoid dependancy on sqlstore.SQLStore itself, so GetDialect could be part of DB.db interface. I got the same issue with team SQLStore Split, not finished yet... https://github.com/grafana/grafana/pull/48883/files#diff-b8f766f730a899b5bc927a021c6a4f4847fa5a9ef26ae60ee2f694199b2835f0R13